api/medical/views.py

```diff
@@ -306,26 +307,28 @@ def post(self, request):
         return Response(AppointmentSerializer(instance).data)
 
 
-
 class AppointmentManagerView(APIView):
     permission_classes = [IsMedicalManager]
 
     def get(self, request):
-        appointments = Appointment.objects.filter(status = "IN PROGRESS").all()
+        appointments = Appointment.objects.filter(status="IN PROGRESS").all()
         serializer = AppointmentSerializer(appointments, many=True)
         return Response(serializer.data)
 
+
 class AppointmentsPending(APIView):
     permission_classes = [IsMedicalManager]
 
     def get(self, request):
         now = datetime.datetime.now()
 
-        appointments = Appointment.objects.filter(status = "ACCEPTED", date__gt=now.date()).all()
+        appointments = Appointment.objects.filter(
+            status="ACCEPTED", date__gt=now.date()).all()
```

Comment:
don't push changes that only change the indentation.

Reply:
Done